src/cdk-experimental/listbox/listbox.ts

```diff
@@ -28,4 +71,23 @@ export class CdkOption {
 })
 export class CdkListbox {
 
+  /** A query list containing all CdkOption elements within this listbox */
+  @ContentChildren(CdkOption, {descendants: true}) _options: QueryList<CdkOption>;
+
+  @Output() readonly selectionChange: EventEmitter<CdkOption> = new EventEmitter<CdkOption>();
```

Comment:
I forgot if I mentioned this earlier, but in a follow-up we'll want to change this from emitting `CdkOption` to emitting an event interface (something like `ListboxSelectionChangeEvent`)

Reply:
Alright sounds good.

Reply:
I looked into it and it seemed pretty straightforward so I rolled it into a commit in this PR. If I didn't implement it right or you think it should go in separately I can remove it.